### PR TITLE
fix style=form examples (3.0.4)

### PR DESCRIPTION
**Pull request by karenetheridge**
These should have the parameter name in the resulting string (see issue #3737).

### CODE REVIEW

**Comment by karenetheridge** [comment]
This will need me to resolve conflicts resolved after merging #3744.

**Review by ralfhandl — APPROVED** [review]

**Review by handrews — DISMISSED** [review]

File: versions/3.0.4.md
Comment (mikekistler) [inline versions/3.0.4.md:1080]: Should this have the name too?
```suggestion
form | true | color= | color=blue | color=blue&color=black&color=brown | color=R=100&G=200&B=150
```

Comment (mikekistler) [reply versions/3.0.4.md:1080]: Also, the lines above, which I cannot comment on, seems to be missing parameter name in the "object" column.

[`style`](#dataTypeFormat) | `explode` | `empty` | `string` | `array` | `object`
----------- | ------ | -------- | -------- | -------- | -------
matrix | true | ;color | ;color=blue | ;color=blue;color=black;color=brown | ;color=R=100;G=200;B=150

**Review by mikekistler — COMMENTED** [review]
Are there other fixes needed here?

File: versions/3.0.4.md
Comment (ralfhandl) [reply versions/3.0.4.md:1080]: I think objects are "exploded" by using their members as query parameter names, see [Parameter Serialization Control](https://learn.openapis.org/specification/parameters.html#parameter-serialization-control), so the current text is what I would expect.

**Comment by ralfhandl** [comment]
@karenetheridge please resolve conflict

**Comment by handrews** [comment]
@ralfhandl @mikekistler @miqui @karenetheridge I resolved the conflicts but that invalidated the approval - can someone re-approve and merge?

**Review by handrews — APPROVED** [review]

**Comment by handrews** [comment]
Oh weird, it let me approve it?  We might want to look into the permissions here, but for now I will merge since there were already 2 TSC approvals and the 3.1.1 and 3.2.0 versions were merged.

**Comment by ralfhandl** [comment]
@handrews you are not the PR author, so you can approve.
We technically require only one approval for merging, and you are a maintainer, so you can approve and then directly merge as long as no other maintainer has requested changes.

"Two TSC approvals" is currently only a convention, not technically enforced.

**Pull request merged**
